frontend/webpack.config.js

```diff
@@ -22,6 +39,23 @@ async function getHttpsOptions() {
 	};
 }
 
+// Extract VITE_PUBLIC_* variables to expose them in the browser
+function getPublicEnvVariables() {
+	const envVars = {};
+	const prefix = 'VITE_PUBLIC_';
+
+	for (const [key, value] of Object.entries(process.env)) {
```

Comment:
This runs at build time, but `.env` gets injected at *run*time, so this probably won't work (see my previous comment).